webcompat/views.py

```diff
@@ -455,6 +455,15 @@ def contributors():
     return render_template('contributors.html')
 
 
+@app.route('/contributors/alumni')
+@cache_policy(private=True, uri_max_age=0, must_revalidate=True)
+def alumni():
+    """Route to display contributors alumni page."""
+    if g.user:
```

Comment:
Not to self, this should just be a decorator. No need to change anything here, but silly to repeat ourselves so much in these routes (something I added, I'm sure).

Reply:
yes! to the refactoring of g.user
Some context https://flask.palletsprojects.com/en/1.1.x/patterns/viewdecorators/

Reply:
Filed #3329